ios/testing/BUILD.gn

```diff
@@ -4,30 +4,34 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import("//build/config/ios/rules.gni")
+import("//ios/build/config.gni")
 import("//testing/test.gni")
 
-ios_xctest_test("brave_core_ios_tests") {
+test("brave_unit_tests") {
```

Comment:
this is confusing. I don't think we should ever use the same name for absolutely different test suites.

`brave_ios_unit_tests` seems fine though.